travis-ci/managers/ubuntu-build-check.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
could you add `set -x` to make it easier to figure out what's going on? I think that should be OK as long as we don't use any credentials here.